packages/headless/src/controllers/product-listing-v2/result-list/headless-product-listing-v2-interactive-result.ts

```diff
@@ -0,0 +1,58 @@
+import {ProductRecommendation} from '../../../api/search/search/product-recommendation';
```

Comment:
We could potentially rename the controller to "InteractiveProduct" or something.

But technically, products are query results, so I'm not opposed to using the generic name.